.nvmrc

```diff
@@ -0,0 +1 @@
+v16.19.1
```

Comment:
We're currently [using 16.15.0](https://github.com/kiva/marketplace-web-ui-ci/blob/main/resources/org/kiva/marketplaceWebUiCi/ui/Dockerfile#L2) but it would be good to match these up with the latest version... so prob fine here to set this value so long as we update the linked dockerfile and verify it too.

Reply:
It actually looks like 16.20.0 is out now too..

Reply:
As an aside... I've been running 16.17.0 locally...

Reply:
Looks like I'm running 16.15 to match docker...

Reply:
Looks like @mcstover just updated ui to 16.20

Reply:
I'm proposing we updated to the latest version and have linked the PR to update the Dockerfile for our Ui images below.